esmvaltool/cmorizers/data/formatters/datasets/lai3g.py

```diff
@@ -19,7 +19,6 @@
     section in the configuration file (`LAI3g.yml`). Note that in this case,
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Fixed.